treefmt.toml

```diff
@@ -0,0 +1,3 @@
+[formatter.nix]
```

Comment:
FYI: Leveraging https://github.com/numtide/treefmt with the hopes of being able to add additional stanzas for https://github.com/numtide/treefmt/wiki#shell) and python via the black formatter (https://github.com/numtide/treefmt/wiki#python)